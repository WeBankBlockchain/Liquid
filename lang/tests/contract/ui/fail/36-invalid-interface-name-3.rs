use liquid_lang as liquid;

#[liquid::interface(name = inherited)]
mod foo {
    extern "liquid" {
        fn bar();
    }
}

fn main() {}