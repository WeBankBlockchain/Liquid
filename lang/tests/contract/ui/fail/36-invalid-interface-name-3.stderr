error: invalid interface name, please specified name as `auto` or a literal string
 --> $DIR/36-invalid-interface-name-3.rs:3:21
  |
3 | #[liquid::interface(name = inherited)]
  |                     ^^^^
